### PR TITLE
feat: convert exchange modals flow in drawers

### PR DESCRIPTION
**Pull request by sarneijim**
### 📝 Description

AC1: Identify all modals currently in use within the swap native flow.

AC2: Design drawer-style interfaces to replace each of these modals in alignment with the style and user interaction flow.

AC3: Implement these design changes across the swap native flow, ensuring that all interactions are now performed through these new drawers.

### ❓ Context

- **Impacted projects**: LLD
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-9424

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

image

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by Justkant — APPROVED** [review]
LGTM, just one question about the margin change

File: apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx [hunk 1]
```diff
@@ -925,7 +932,7 @@ export const renderSwapDeviceConfirmation = ({
             />
           </Alert>
         </Box>
-        <Box mx={6} data-test-id="device-swap-summary">
+        <Box mx={3} data-test-id="device-swap-summary">
```
[the remark this answers is not in the document]
Comment (sarneijim) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx:935]: Lol, the look field more similar what we have in swap drawer

[the remark this answers is not in the document]
Comment (Justkant) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx:935]: Don't really understand what's funny

**Pull request merged**